docs/track.rst

```diff
@@ -122,6 +123,10 @@ In the ``operations`` section you describe which operations are available for th
 
 In the ``schedule`` section you describe the workload for the benchmark, for example index with two clients at maximum throughput while searching with another two clients with ten operations per second. The schedule either uses the operations defined in the ``operations`` block or defines the operations to execute inline.
 
+In the ``challenges`` section you describe more than one set of operations, in the event your track needs to test than one set of scenarios. If only one set of operations is needed, they can be specified in ``schedule`` instead.
```

Comment:
Two comments:

1) as you figured out, [challenges are optional](https://esrally.readthedocs.io/en/stable/adding_tracks.html?highlight=challenges#challenges); that's why they aren't in this list. I thin here we should just mention them as optional and link to the [docs](https://esrally.readthedocs.io/en/stable/adding_tracks.html?highlight=challenges#challenges) as it also explains when they could be useful.

2) `needs to test than one set of scenarios` -> `needs to test` **more** `than one set of scenarios`

Reply:
So are u saying put a note here but dont include them in the bullet list above?

Reply:
As it's optional, I'd tend to yes, just a note.